src/plugins/global-settings/components/global-settings-color-picker/index.js

```diff
@@ -0,0 +1,472 @@
+/**
```

Comment:
Please move this file into it's own file in `plugins/global-settings/colors/color-picker.js` instead of a new folder inside `plugins/global-settings`